docs/content/2.deploy/20.providers/cloudflare.md

```diff
@@ -10,17 +10,15 @@ Deploy Nitro apps to CloudFlare.
 **Note:** This preset uses [service-worker syntax](https://developers.cloudflare.com/workers/learning/service-worker/) for deployment.
 ::
 
-Login to your [Cloudflare Workers](https://workers.cloudflare.com) account and obtain your `account_id` from the sidebar.
```

Comment:
Developers do not need to have their account id upfront, so I feel like putting this here can just create unnecessary churn, I think it's better to just have it in the toml below where it is clear that it is an optional value